src/openalea/strawberry/import_mtgfile.py

```diff
@@ -36,18 +34,3 @@ def name(f):
         for i in mtgfile:
             metaMTG = algo.union(metaMTG, MTG(mtgfile[i]))
         return metaMTG   
-
-
```

Comment:
Why delete this function?

Reply:
This function do nothing.
There is no print or something else.
So I remove it.